docs/plugins/inputs/github.asciidoc

```diff
@@ -1,14 +1,13 @@
 :plugin: github
 :type: input
 :default_plugin: 0
-:default_codec: plain
 
 ///////////////////////////////////////////
 START - GENERATED VARIABLES, DO NOT EDIT!
 ///////////////////////////////////////////
-:version: v3.0.7
-:release_date: 2018-04-06
-:changelog_url: https://github.com/logstash-plugins/logstash-input-github/blob/v3.0.7/CHANGELOG.md
+:version: v3.0.5
+:release_date: 2017-11-07
+:changelog_url: https://github.com/logstash-plugins/logstash-input-github/blob/v3.0.5/CHANGELOG.md
```

Comment:
Verify that this regression is correct. It removes `:default_codec!:`, and I don't think that's what we want.

Reply:
it is, because a transitive dependency causes a previous version to be installed. we may need to sort this out but it is not blocking for this merge. 
Regarding the codec, this plugin doesn't use codecs. we can introduce the no_codec setting in a separate pr